packages/preload/src/index.ts

```diff
@@ -2,4 +2,10 @@
  * @module preload
  */
 
+import { stopPeriodicScraping } from './eventsBridge';
+
 export * from './eventsBridge';
+
+// Clear the interval that scrapes periodically
+await stopPeriodicScraping();
```

Comment:
I'm not sure about this, wrote you on Whatsapp

Reply:
Removed this